firmware/integration/rusefi_config.txt

```diff
@@ -1131,32 +1136,33 @@ custom pin_mode_e 1 bits, U08, @OFFSET@, [0:6], @@pin_mode_e_enum@@
 	brain_input_pin_e turboSpeedSensorInputPin;
 	float alternator_derivativeFilterLoss;;"x", 1, 0, -1000000, 1000000, 4
 	float alternator_antiwindupFreq;;"x", 1, 0, -1000000, 1000000, 4
-int16_t tps2Min;Closed throttle#2. todo: extract these two fields into a structure\nSee also tps2_1AdcChannel\nset tps2_min X;"ADC", 1, 0, 0, 1023, 0
-int16_t tps2Max;Full throttle#2. tpsMax value as 10 bit ADC value. Not Voltage!\nSee also tps1_1AdcChannel\nset tps2_max X;"ADC", 1, 0, 0, 1023, 0
-	output_pin_e starterControlPin;See also startStopButtonPin
+! todo: extract these two fields into a structure
+int16_t tps2Min;+Closed throttle#2. \nSee also tps2_1AdcChannel\nset tps2_min X;"ADC", 1, 0, 0, 1023, 0
+int16_t tps2Max;+Full throttle#2. tpsMax value as 10 bit ADC value. Not Voltage!\nSee also tps1_1AdcChannel\nset tps2_max X;"ADC", 1, 0, 0, 1023, 0
+	output_pin_e starterControlPin;+See also startStopButtonPin
 	pin_input_mode_e startStopButtonMode;
 	brain_pin_e mc33816_flag0;
 	uint8_t tachPulsePerRev;;"Pulse", 1, 0, 1, 255, 0
 
 ! todo: mapErrorDetectionIdleTooLow? 30kPa is usually lowest on idle	
-	float mapErrorDetectionTooLow;kPa value which is too low to be true;"kPa", 1, 0, -100, 100, 2
-	float mapErrorDetectionTooHigh;kPa value which is too high to be true;"kPa", 1, 0, -100, 800, 2
+	float mapErrorDetectionTooLow;+kPa value which is too low to be true;"kPa", 1, 0, -100, 100, 2
+	float mapErrorDetectionTooHigh;+kPa value which is too high to be true;"kPa", 1, 0, -100, 800, 2
 	uint16_t multisparkSparkDuration;+How long to wait for the spark to fire before recharging the coil for another spark.;"ms", 0.001, 0, 0, 3, 2
 	uint16_t multisparkDwell;+This sets the dwell time for subsequent sparks. The main spark's dwell is set by the dwell table.;"ms", 0.001, 0, 0, 3, 2
-	pid_s idleRpmPid;See cltIdleRpmBins
+	pid_s idleRpmPid;+See cltIdleRpmBins
```

Comment:
tooltips on structs (like this pid_s) don't actually show anywhere, so the `+` does nothing.